cpp/src/io/parquet/reader_impl_chunking.cu

```diff
@@ -386,7 +386,7 @@ int64_t find_next_split(int64_t cur_pos,
     // if decimal but not outputting as float or decimal, then convert to no logical type
     if (column_type_id != type_id::FLOAT64 and
         not cudf::is_fixed_point(data_type{column_type_id})) {
-      return std::make_tuple(clock_rate, cuda::std::nullopt);
+      return std::make_tuple(clock_rate, std::nullopt);
```

Comment:
While we're here, ought we to consider this too?
```suggestion
      return s{clock_rate, std::nullopt};
```

Reply:
Done!